### ISSUE TITLE
[Feature Request] Option to remove the extra letterboxing when Cutscene Subtitles are enabled

### ISSUE THREAD

**Issue by Dreamboum** [fgsfdsfgs/perfect_dark#504]
![564564654](https://github.com/user-attachments/assets/1a1b6bee-0bf6-46f8-83dd-d23e6c1cc64e)

I changed my screen to a 2560x1440, 170 Hz, and since then I've had this black band appearing at the bottom, whereas on my 1080p screen it wasn't doing it to me. So I don't know if it's because of a resolution incompatibility or what.

**Comment by fgsfdsfgs** [comment]
It's always been there. Do you have cutscene subtitles enabled?

**Comment by Dreamboum** [comment]
Yes, I just noticed that I used a save with subtitles enabled, but I was wondering if it was possible to turn off that black band, even though there are subtitles?

**Comment by fgsfdsfgs** [comment]
Not at the moment. That is the background for the subtitles. I'd need to add some sort of drop shadow to them or something and an option to turn off the extra letterboxing.
